app/src/test/java/org/simple/clinic/summary/PatientSummaryScreenControllerTest.kt

```diff
@@ -417,6 +416,55 @@ class PatientSummaryScreenControllerTest {
       arrayOf(false, PatientSummaryCaller.SEARCH)
   )
 
+  @Test
+  @Parameters(value = ["SEARCH", "NEW_PATIENT"])
+  fun `when all BPs for the patient are deleted and back is clicked, the schedule appointment sheet must not be shown`(
+      patientSummaryCaller: PatientSummaryCaller
+  ) {
+    whenever(bpRepository.bloodPressureCount(patientUuid)).thenReturn(Observable.just(0))
+
+    uiEvents.onNext(PatientSummaryScreenCreated(patientUuid, caller = patientSummaryCaller, screenCreatedTimestamp = Instant.now(clock)))
+    uiEvents.onNext(PatientSummaryBloodPressureClosed(true))
```

Comment:
Can we also pass `wasBloodPressureSaved` in the params?

Reply:
Done.